docs/conf.py.jinja

```diff
@@ -55,6 +55,7 @@ intersphinx_mapping = {
     'python': ('https://docs.python.org/3', None),
     'numpy': ('https://numpy.org/doc/stable/', None),
 }
+intersphinx_mapping.pop('{{projectname}}', None)
```

Comment:
What is this doing?

Reply:
I don't really understand the question. I obviously just forgot to remove it.